projects/batfish/src/main/java/org/batfish/main/Batfish.java

```diff
@@ -4167,7 +4167,9 @@ public AnswerElement standard(
       String ingressNodeRegexStr,
       String notIngressNodeRegexStr,
       String finalNodeRegexStr,
-      String notFinalNodeRegexStr) {
+      String notFinalNodeRegexStr,
+      Set<String> transitNodes,
+      Set<String> notTransitNodes) {
```

Comment:
If these are allowed to be null, annotate `@Nullable`. Either way, when stored they should not be allowed to be null